src/pages/settings/InitialSettingsPage.js

```diff
@@ -52,6 +53,9 @@ const propTypes = {
 
         /** The user's role in the policy */
         role: PropTypes.string,
+
+        /** The current action that is waiting to happen on the policy */
+        pendingAction: PropTypes.string,
```

Comment:
Let's update this to be more precise `pendingAction: PropTypes.oneOf(['add', 'update', 'delete']),`

Reply:
+1

Reply:
+1

Reply:
Updated!